doc/api/util.md

```diff
@@ -3287,6 +3319,8 @@ util.log('Timestamped message.');
 [Customizing `util.inspect` colors]: #customizing-utilinspect-colors
 [Internationalization]: intl.md
 [Module Namespace Object]: https://tc39.github.io/ecma262/#sec-module-namespace-exotic-objects
+[realm]: https://tc39.es/ecma262/#realm
+[built-in `Error` type]: https://tc39.es/ecma262/#sec-error-objects
```

Comment:
Can you fix the ordering here please? All references should appear in ASCII order.

Reply:
Done!